### PR TITLE
* citre-basic-tools.el (citre-create-tags-file): update read-char-cho…

### PR DESCRIPTION
**Pull request by tumashu**
…ice prompt.

### CODE REVIEW

**Comment by tumashu** [comment]
```
Citre save tags file.
[1] Save to a directory specified.
[2] Save to global cache directory.
[3] Save to project cache directory.
[4] Save to a file specified (modify `citre-tags-file-alist' is required).
==> Please type a number (1-4) to choose: 
```
个人感觉这个 prompt 可能更漂亮一点 :-)

**Comment by AmaiKinono** [comment]
Thanks! However, I do think some of the words are not clear as before. What do you think of:

```
Save tags file to...
[1] the directory where I want to use it.
[2] global cache directory.
[3] project cache directory.
[4] I'll choose a file myself (modify `citre-tags-file-alist' is then required to use it).
==> Please type a number (1-4) to choose: 
```

**Comment by tumashu** [comment]
@AmaiKinono  看起来也不错，pr 更改了

File: citre-basic-tools.el [hunk 1]
```diff
@@ -607,14 +607,12 @@ An updating recipe is written to it so later it can be updated by
...
-[3] In the project cache dir
-[4] I'll select a file myself
-NOTE: If you choose 4, you need to modify `citre-tags-file-alist' \
-for it to to be used in a dir
-"
+                  (pcase (read-char-choice "Save tags file to ...
+[1] the directory where I want to use it.
+[2] global cache directory.
+[3] project cache directory.
+[4] I'll choose a file myself (modify `citre-tags-file-alist' is then required to use it).
```
Comment (AmaiKinono) [inline citre-basic-tools.el:614]:
```suggestion
[4] I'll choose a file myself (modify `citre-tags-file-alist' is then \
required to use it).
```

Tip: You can run `$ make` to run the tests locally. `$ make style` to check the style.

Comment (tumashu) [reply citre-basic-tools.el]: should be fixed.

**Comment by AmaiKinono** [comment]
Thanks!

**Pull request merged**